packages/shared/providers.yaml

```diff
@@ -413,7 +413,7 @@ canny:
     proxy:
         base_url: https://canny.io/api/v1
         query:
-            apiKey: ${apiKey}
+            api_key: ${apiKey}
```

Comment:
@bodinsamuel , this might break for users using canny, as the provider requires this specific [field](https://developers.canny.io/api-reference#intro): `apiKey=YOUR_API_KEY`

Reply:
ah I see it's a free form thanks 👌🏻